src/main/java/io/confluent/kafkarest/SimpleConsumerManager.java

```diff
@@ -64,13 +49,13 @@
   private final MetadataObserver mdObserver;
   private final SimpleConsumerFactory simpleConsumerFactory;
```

Comment:
Seems we don't need this anymore since the factory was moved into SimpleConsumerPool.